config/jobs/image-pushing/k8s-staging-etcd-manager.yaml

```diff
@@ -22,8 +22,8 @@ postsubmits:
               - /run.sh
             args:
               # this is the project GCB will run in, which is the same as the GCR images are pushed to.
-              - --project=k8s-staging-images
-              - --scratch-bucket=gs://k8s-staging-images-gcb
+              - --project=k8s-staging-etcd-manager
+              - --scratch-bucket=gs://k8s-staging-etcd-manager-gcb
```

Comment:
Although this bucket does not seem to exist...

Reply:
Turnout we no longer can create GCR repositories with the `./ensure-staging-storage.sh` script. They are now hosted on AR. https://cloud.google.com/container-registry/docs/deprecations/container-registry-deprecation